R/pkg/R/mllib_fpm.R

```diff
@@ -183,8 +183,8 @@ setMethod("write.ml", signature(object = "FPGrowthModel", path = "character"),
 #' @return A complete set of frequent sequential patterns in the input sequences of itemsets.
 #'         The returned \code{SparkDataFrame} contains columns of sequence and corresponding
 #'         frequency. The schema of it will be:
-#'         \code{sequence: ArrayType(ArrayType(T))} (T is the item type)
-#'         \code{freq: Long}
+#'         \code{sequence: ArrayType(ArrayType(T))}, \code{freq: integer}
+#'         where T is the item type
```

Comment:
I know there is a problem here too. But the one that needs to get fixed is in line 637 in R/pkg/R/mllib_clustering.R. 
I guess I will have a separate follow up PR to fix the mllib_fpm.R problem. Sorry for the mess.

Reply:
Yeah I wanted to fix both here while we're at it.

Reply:
let's fix both